File: packages/SystemUI/src/com/google/android/systemui/smartspace/DefaultBcSmartspaceConfigProvider.java
package com.google.android.systemui.smartspace;

import com.android.systemui.plugins.BcSmartspaceConfigPlugin;

public final class DefaultBcSmartspaceConfigProvider implements BcSmartspaceConfigPlugin {
    @Override
    public final boolean isDefaultDateWeatherDisabled() {
        return false;
    }

    @Override
    public final boolean isSwipeEventLoggingEnabled() {
        return false;
    }

    @Override
    public final boolean isViewPager2Enabled() {
        return false;
    }
}

File: packages/SystemUI/src/com/google/android/systemui/smartspace/DefaultBcSmartspaceConfigProvider.kt
package com.google.android.systemui.smartspace

import com.android.systemui.plugins.BcSmartspaceConfigPlugin

class DefaultBcSmartspaceConfigProvider : BcSmartspaceConfigPlugin {
    override val isDefaultDateWeatherDisabled: Boolean
        get() = true

    override val isViewPager2Enabled: Boolean
        get() = false

    override val isSwipeEventLoggingEnabled: Boolean
        get() = false
}
